nilearn/decoding/searchlight.py

```diff
@@ -23,7 +23,7 @@
 
 from .. import masking
 from ..image.resampling import coord_transform
-from ..input_data.nifti_spheres_masker import _apply_mask_and_get_affinity
+from nilearn.maskers.nifti_spheres_masker import _apply_mask_and_get_affinity
```

Comment:
for example this import and the one in decoder.py wouldn't work anymore with the current interface